amundsen_application/api/mail/v0.py

```diff
@@ -75,3 +75,14 @@ def _feedback(*,
               subject: str) -> None:
     """ Logs the content of the feedback form """
     pass  # pragma: no cover
+
+
+@mail_blueprint.route('/notification', methods=['POST'])
+def notification() -> Response:
```

Comment:
Something that I find very helpful for partial/dummy implementations is to leave "TODO" comments in the code. This helps me not nitpick about pieces of implementation that will be able to be addressed later. It will also serve a reminder to you to not forget to come back to certain things.

For example a "TODO" that could be added in the try block would be "When making a real api call, check the status code and return appropriate response". Example of what I mean:
https://github.com/lyft/amundsenfrontendlibrary/blob/master/amundsen_application/api/mail/v0.py#L50-L59